lib/diaspora_federation/validators/relayable_validator.rb

```diff
@@ -0,0 +1,11 @@
+module DiasporaFederation
+  module Validators
+    module RelayableValidator
+      def self.included(model)
+        model.class_eval do
+          rule :parent_guid, :guid
```

Comment:
But I think at least `author_signature` should be there or the signature-verification will fail?

I don't know where the `parent_author_signature` is used and what happens if it is empty.